glide.lock

```diff
@@ -223,7 +223,7 @@ imports:
   - transport
 - name: gopkg.in/src-d/go-errors.v0
   version: 0c303ec4c027302259ec1738f19f515aad25f13f
-- name: gopkg.in/src-d/simple-linguist.v1
+- name: gopkg.in/src-d/enry.v1
```

Comment:
@ajnavarro that's right. But our internal imports use `gopkg.in/src-d/enry.v1` because we were not using glide before and we needed some way to import a particular version.  Now that we are using glide, that should no longer be the case.

What I am talking about is to start importing `github.com/src-d/enry` everywhere, now that thanks to glide, we no longer need to use URL redirects to simulate versioning.